adal/src/main/java/com/microsoft/aad/adal/AuthenticationConstants.java

```diff
@@ -451,7 +451,7 @@ private AuthenticationConstants() { }
          * Signature info for Azure authenticator app that installs authenticator
          * component.
          */
-        public static final String AZURE_AUTHENTICATOR_APP_SIGNATURE = "ho040S3ffZkmxqtQrSwpTVOn9r0=";
```

Comment:
@heidijinxujia Right. Although it should be noted that msauth automation won't work without this change.